.spellcheck.yaml

```
@@ -13,7 +13,7 @@ matrix:
       context_visible_first: true
       delimiters:
       # ReST code blocks
-      - open: '^.{2} code-block'
+      - open: '^.{2}\scode-block'
```

We should not accept _any_ whitespace character.
```suggestion
      - open: '^.{2}[ ]+code-block'
```